src/main/java/uk/nhs/digital/nhsconnect/nhais/mapper/AcceptanceTypeMapper.java

```diff
@@ -0,0 +1,25 @@
+package uk.nhs.digital.nhsconnect.nhais.mapper;
+
+import org.hl7.fhir.r4.model.Parameters;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.AcceptanceType;
+
+public class AcceptanceTypeMapper implements FromFhirToEdifactMapper<AcceptanceType> {
+    private final static String ACCEPTANCE_TYPE = "acceptanceType";
+
+    public AcceptanceType map(Parameters parameters) {
+        return AcceptanceType.builder()
+            .type(getAcceptanceType(parameters))
+            .build();
+    }
+
+    private String getAcceptanceType(Parameters parameters) {
+        return parameters.getParameter()
+            .stream()
+            .filter(param -> ACCEPTANCE_TYPE.equals(param.getName()))
+            .map(Parameters.ParametersParameterComponent::getValue)
+            .map(Object::toString)
+            .map(AcceptanceType::getTypeValue)
+            .findFirst()
+            .orElseThrow(() -> new IllegalStateException("Acceptance Type not supported"));
```

Comment:
Please throw a FhirValidationException and identify which parameter is missing (by name) so that a correct OperationOutcome response is produced by the API